server/src/instant/util/exception.clj

```diff
@@ -46,6 +47,24 @@
 (s/def ::trace-id string?)
 (s/def ::instant-exception (s/keys :req [::type ::message ::trace-id]))
 
+(def bad-request-types #{::record-not-found
```

Comment:
Moved these from `instant.util.http` so that they'll be easier to update when you update the set of `::types`